### PR TITLE
Bedrock genai demo

### PR DESCRIPTION
**Pull request by mxiamxia**
*Description of changes:*
Add Bedrock & Bedrock runtime API call examples for feel and trial purpose. The major changes are
1. Added a new `customer/diagnose/owners/*` API in gateway service for Bedrock api invokes'
2. Extended customer service to
   * call BedrockAgent via AWS SDK v1 and v2
   * call Bedrock runtime via AWS SDK v1 on Titian model and v2 on Claude model
   * call bedrock via AWS SDK v1 and v2 for Guardrails APIs

image

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: spring-petclinic-customers-service/src/main/java/org/springframework/samples/petclinic/customers/aws/BedrockRuntimeV2Service.java [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+                    .contentType("application/json")
+                    .accept("application/json")
+                    .build();
+            System.out.println("Anthropic claude request:" + request.body().asUtf8String());
+
+            InvokeModelResponse response = bedrockRuntimeV2Client.invokeModel(request);
+
+            JSONObject responseBody = new JSONObject(response.body().asUtf8String());
+            System.out.println("invokeLlama2 response:" + response.body().asUtf8String());
+            // String generatedText = responseBody.getString("generation");
```
Comment (jyothimc) [inline spring-petclinic-customers-service/src/main/java/org/springframework/samples/petclinic/customers/aws/BedrockRuntimeV2Service.java:68]: This is my bad from copying over the code. Are you able to please uncomment these?

File: spring-petclinic-customers-service/src/main/java/org/springframework/samples/petclinic/customers/aws/BedrockRuntimeV2Service.java [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+                    .build();
+        }
+
+    }
+
+    public String invokeAnthropicClaude() {
+        try {
+            System.out.printf("Invoke Anthropic claude: ");
+            String claudeModelId = "anthropic.claude-3-sonnet-20240229-v1:0";
+            String prompt = "What's the common desease for a pet?";
```
Comment (jyothimc) [inline spring-petclinic-customers-service/src/main/java/org/springframework/samples/petclinic/customers/aws/BedrockRuntimeV2Service.java:39]: Again - my bad from copying over - typo - "disease"

File: spring-petclinic-customers-service/src/main/java/org/springframework/samples/petclinic/customers/aws/BedrockRuntimeV1Service.java [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+                    .build();
+        }
+
+    }
+
+    public String invokeTitanModel() {
+        try {
+            System.out.printf("invokeTitanModel: ");
+            String modelId = "amazon.titan-text-express-v1";
+            String inputText = "What's the common desease for a pet?";
```
Comment (jyothimc) [inline spring-petclinic-customers-service/src/main/java/org/springframework/samples/petclinic/customers/aws/BedrockRuntimeV1Service.java:44]: My bad - typo 'disease'

**Review by jyothimc — CHANGES_REQUESTED** [review]
Some minor comments - mostly from my copy paste

**Review by pxaws — APPROVED** [review]

**Pull request merged**